modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/GridDhtLocalPartition.java

```diff
@@ -167,6 +170,9 @@
     /** */
     private volatile long clearVer;
 
+    /** */
+    private volatile long reclearVer;
```

Comment:
Is it possible to reuse `clearVer` instead of a new field?

Reply:
Fixed